.github/workflows/build.yml

```diff
@@ -55,13 +55,36 @@ jobs:
             dist/*.tar.gz
             dist/*.zip
 
-      - name: Generate Installers
-        run: |
-          mkdir -p dist-installers
-          ./scripts/generate-installers.sh ./dist-installers
+  installer:
+    name: Windows installers
+    needs: [ build ]
+    runs-on: windows-2022
 
-      - name: Upload Installers
-        uses: actions/upload-artifact@v4
-        with:
-          name: backrest-snapshot-installers
-          path: dist-installers/*.exe
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Download build artifacts
+      uses: actions/download-artifact@v4
+      with:
+        name: backrest-snapshot-builds
+
+    - name: Unzip artifacts and compile Inno Setup installers
+      shell: powershell
```

Comment:
nice, I think overall switching to and actually using windows/powershell to build the windows installer is a nice touch and makes sense.